assignments/callbacks.js

```diff
@@ -1,29 +1,50 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+function print (output){
+  console.log(output);
+}
+
+
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+ return cb(arr[0]);
 }
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x+y);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x *y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+for(let i = 0; i <  list.length; i++){
+if (list[i] === item){
+  return true;
+}
+else {
+  return false;
 }
```

Comment:
So this method will not actually work for contains. There are two possible scenarios:
1. The element you are looking for is the first element in the array, so your method returns true
2. The element you are looking for is either in the array elsewhere, or isn't in the array and  your method returns false

The reason for this is the structure of your if statement. Essentially as soon as you run the if statement one time, one of the returns will fire and the function will be exited. If you move the `return false` out of your for loop, it should fix the problem.